lib/cylc/cfgspec/globalcfg.py

```diff
@@ -189,12 +191,13 @@
             'global init-script': vdr(vtype='string'),
             'copyable environment variables': vdr(
                 vtype='string_list', default=[]),
-            'retrieve job logs': vdr(vtype='boolean', default=False),
+            'retrieve job logs': vdr(vtype='boolean'),
```

Comment:
Probably obvious and I'm just not seeing it, but why no default any more for this and the other entries?